src/probabilities_estimators/permutation_ordinal/SymbolicPermutation.jl

```diff
@@ -76,9 +68,9 @@ information about within-state-vector amplitudes.
     `lt = Base.isless`).
 
 [^BandtPompe2002]: Bandt, Christoph, and Bernd Pompe. "Permutation entropy: a natural
-    complexity measure for time series." Physical review letters 88.17 (2002): 174102.
+    complexity measure for timeseries." Physical review letters 88.17 (2002): 174102.
 [^Zunino2017]: Zunino, L., Olivares, F., Scholkmann, F., & Rosso, O. A. (2017).
-    Permutation entropy based time series analysis: Equalities in the input signal can
+    Permutation entropy based timeseries analysis: Equalities in the input signal can
```

Comment:
This is actually not correct. The title of the paper splits "timeseries" into "time series". 

Which begs the question. Why are we spelling it as "timeseries" and not "time series"? The spelling "timeseries" gives 2.5 million search hits on google, while "time series" gives 160 million hits. Moreover, both the Oxford and Merriam-Webster dictionaries redirect to "time series" (with a space). Alternative spellings are given as time-series.

Reply:
The google argument isn't particularly strong though because two word searches serach the union of {"time", "series", "time series"} which is a big superset over just "time series". So:

![image](https://user-images.githubusercontent.com/19669089/209197266-447ca02d-491d-4894-8684-69c212643c9e.png)

![image](https://user-images.githubusercontent.com/19669089/209197332-8e31d583-0129-4e7d-81ca-749e868ff795.png)

I've used exclusively timeseries (one word) in all other docs of JuliaDynamics I write, and in the NLD textbook. My reasoning is that a timeseries is a very specific thing that is more specific than just "time" and "series" together. So might as well make them one word. I am not 100% sold on this or anything, but I've used the single word in so many other JuliaDynamics words that would be a pain to revert.

The dicctionary arguments don't matter for me personally because here we are talking about a field specific term, the field of complex systems.

Reply:
I'm usually pedantic about keeping the English spelling, so for the combination, I'd use time-series instead of "timeseries".  But might as well just set the standard here. Refactoring across an entire ecosystem of packages is pointless, and arguing over wording which have multiple valid spellings are even more pointless :D